api/src/main/resources/application.yaml

```diff
@@ -163,6 +127,16 @@ batch:
     cron: ${CRON_SCHEDULED_PURGE_OLD_RECORDS}
     staleInDays: ${RECORDS_STALE_IN_DAYS}
 
+#Resilience
+resilience:
+  retry:
+    default:
+      maxAttempts: ${MAX_RETRY_ATTEMPTS}
+      waitDuration: 2
+    get-token:
+      maxAttempts: 35
```

Comment:
I think we can do with way less retries for token. 35 attempts at 5 sec intervals means 3 mins on connection. Connection would probably terminate. Maybe just try 5 attempts at 2 seconds.

Reply:
Good catch!  I totally agree to change retry for token.   It is committed!